app/src/main/java/com/swent/assos/ui/login/LoginScreen.kt

```diff
@@ -23,11 +23,12 @@ import androidx.compose.ui.text.input.PasswordVisualTransformation
 import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavHostController
 import com.swent.assos.R
 import com.swent.assos.model.view.LoginViewModel
 
 @Composable
-fun LoginScreen() {
+fun LoginScreen(navController: NavHostController) {
```

Comment:
replace `navController` by an argument with type `NavigationActions`. It is better for encapsulation (we limit the access to the NavHostController)